packages/daemon/src/config/config.ts

```diff
@@ -0,0 +1,11 @@
+import { homedir } from 'os';
+
+
+const  dialect = 'sqlite';
+const storage = process.env.PIPCOOK_STORAGE || (`${homedir()}/.pipcook/db/pipcook.db`)
```

Comment:
Could we use https://github.com/alibaba/pipcook/blob/master/packages/core/src/constants/other.ts#L87?